### DISCUSSION TITLE
If a figure is a concave polygon, judge whether it intersects, the result is wrong

### DISCUSSION THREAD

**Discussion by Lynne42** [fabricjs/fabric.js#7215] [Q&A]
If a figure is a concave polygon, judge whether it intersects, the result is wrong

https://codepen.io/qlniu/pen/gOWLaOe?editors=1111

![image](https://user-images.githubusercontent.com/46595240/125759337-bf376ab8-4f54-48e0-bdcd-ddee315c1e1d.png)

**Comment by asturur** [comment]
@qlniu our intersect with object is just for bounding boxes.
